weekly-mission(week2)/src/apis/signInUser.js

```diff
@@ -0,0 +1,22 @@
+// 로그인 관련 통신 함수들 모음
+
+const signIn = async (inputAccount) => {
```

Comment:
signIn 함수의 입장에서만 보면 매개변수로 계정에 대한 데이터를 받으면 되는데요~ 이 데이터가 어디서 왔는지(input)에 대한 맥락은 사실 필요하지 않다고 보는게 맞을 것 같아요!
따라서 account만 적어주는게 좋을 것 같아요~
```suggestion
const signIn = async (account) => {
```

Reply:
혹은 요런 느낌도 괜찮습니다!
```suggestion
const signIn = async ({ email, password }) => {
```